composer.json

```diff
@@ -125,6 +128,7 @@
         "drupal/metatag": "^1.0",
         "drupal/paragraphs": "1.3",
         "drupal/paragraphs_features": "^1.0",
+        "drupal/password_policy": "^3.0",
```

Comment:
We are using a patch. So this should be a fixed version

Reply:
Also the module is not added to the makefile. Or should we just add this module to the composer suggestion?

Reply:
Good idea, but as you said, when using a patch the version should be fixed, with suggestions, we cannot do that.
I will add it to the makefile as well for now.